services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models import QuerySet
+from db.models import User
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> None:
+    user = get_user_model().objects.create_user(username=username)
+    if password:
+        user.set_password(password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> QuerySet:
+    queryset = User.objects.get(id=user_id)

```

Comment:
Use `get_user_model()` instead of `User`, it is the best practice.
it is the 4th item on checklist:
https://github.com/mate-academy/py-tickets-and-orders/blob/master/checklist.md#code-style